.travis.yml

```diff
@@ -22,8 +22,78 @@ before_script:
   - source ci/setup_env.sh
   - ci/travis_keep_alive.sh
 
+cache:
+  directories:
+    - $CACHE_DIR
+
+.com.github.horizenofficial.sidechain-sdk.job-definitions:
+  - &test_docker_amd64_x-large
+    if: NOT env(SKIP_TESTS)
+    os: linux
+    dist: focal
+    vm:
+      size: x-large
+    script:
+      - bash -c "chmod +x ${TRAVIS_BUILD_DIR}/ci/run_python_tests.sh"
+      - bash -c "${TRAVIS_BUILD_DIR}/ci/run_python_tests.sh ${TEST_CMD} ${TEST_ARGS} ${NODE_VERSION}"
+
 jobs:
+  fast_finish: true
   include:
+    - stage: Test
+      <<: *test_docker_amd64_x-large
+      env:
+        - NODE_VERSION="v16.0.0"
+        - TEST_CMD="./run_sc_tests.sh"
+        - TEST_ARGS="-split=9:1"
+    - stage: Test
+      <<: *test_docker_amd64_x-large
+      env:
+        - NODE_VERSION="v16.0.0"
```

Comment:
Can we add this to globla_env block or need it per job basis?

Reply:
Also better to just use `16` as the value for `nvm install`, that will always install the latest version of node 16 with potential security relevant fixes instead of hard coded `v16.0.0`.